docs/src/pages/components/data-grid/filtering/filtering.md

```diff
@@ -24,6 +24,12 @@ In addition to the column menu that allows users to apply a filter, you can also
 
 {{"demo": "pages/components/data-grid/filtering/BasicToolbarFilteringGrid.js", "bg": "inline"}}
 
+### Filter Operators
+
+By default the string operator is used for filtering. Additionally numerical and date operators are available:
+
+{{"demo": "pages/components/data-grid/filtering/FilterOperators.js", "bg": "inline"}}
```

Comment:
I don't fully get this comment, if I add the proposed option it would automatically hide the code - is this something we want? In all the other examples it seems it's expanded by default.

Reply:
There are two "codes". One that is inlined, one that has to be expended. This hides the inlined one.

Reply:
Ah okay, now I get it, as in the inline you can't see anything of help as it just shows the render part. Changed 👍